### ISSUE TITLE
Can we get 3.10 wheels? 

### ISSUE THREAD

**Issue by matthewdeanmartin** [ionelmc/python-lazy-object-proxy#60]
Their absence creates subtle python 3.10 incompatibilities with tools like pylint.
```
pipenv install --python=3.10 --skip-lock
pipenv shell
# mitigate supply chain risk by using --only-binary
pip install pylint --only-binary=:all:
```
fails because python-lazy-object-proxy [lacks 3.10 wheels](https://pypi.org/project/lazy-object-proxy/#files)

Thanks for maintaining the project!

**Comment by ionelmc** [comment]
It's on my todo list, but it's gonna take few days to switch to github actions as travis has effectively killed opensource support. I'll also have to drop 2.7 support - so far I've found out it's near impossible to build 2.7 on gha.

**Comment by matthewdeanmartin** [comment]
We'll be patient! Good riddance to 2.7! Almost makes me want to go around asking projects to yank their 2.7 wheels, ha!

**Comment by ionelmc** [comment]
Won't yank anything lol, I don't do that kind of nasty :)) There will be a new LOP release.

**Comment by ionelmc** [comment]
Released 1.7.0.
